tests/quasi-newton/serial/TestQN11.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<precice-configuration>
+  <data:scalar name="Data1" />
+  <data:scalar name="Data2" />
+
+  <mesh name="MeshOne" dimensions="2">
+    <use-data name="Data1" />
+    <use-data name="Data2" />
+  </mesh>
+
+  <mesh name="MeshTwo" dimensions="2">
+    <use-data name="Data1" />
+    <use-data name="Data2" />
+  </mesh>
+
+  <participant name="SolverOne">
+    <provide-mesh name="MeshOne" />
+    <write-data name="Data1" mesh="MeshOne" />
+    <read-data name="Data2" mesh="MeshOne" />
+  </participant>
+
+  <participant name="SolverTwo">
+    <receive-mesh name="MeshOne" from="SolverOne" safety-factor="0.1" />
+    <provide-mesh name="MeshTwo" />
+    <mapping:nearest-neighbor
+      direction="read"
+      from="MeshOne"
+      to="MeshTwo"
+      constraint="consistent" />
+    <mapping:nearest-neighbor
+      direction="write"
+      from="MeshTwo"
+      to="MeshOne"
+      constraint="conservative" />
+    <write-data name="Data2" mesh="MeshTwo" />
+    <read-data name="Data1" mesh="MeshTwo" />
+  </participant>
+
+  <m2n:sockets acceptor="SolverOne" connector="SolverTwo" />
+
+  <coupling-scheme:serial-implicit>
+    <participants first="SolverOne" second="SolverTwo" />
+    <max-time-windows value="2" />
+    <time-window-size value="1.0" />
+    <exchange data="Data1" mesh="MeshOne" from="SolverOne" to="SolverTwo" />
+    <exchange data="Data2" mesh="MeshOne" from="SolverTwo" to="SolverOne" />
+    <max-iterations value="200" />
+    <absolute-convergence-measure limit="1e-8" data="Data1" mesh="MeshOne" />
+    <acceleration:IQN-IMVJ>
+      <data name="Data2" mesh="MeshOne" lower-bound="-1" upper-bound="1" />
```

Comment:
Could it also make sense to limit [secondary data](https://precice.org/configuration-acceleration.html#coupling-data-and-primary-data)? Then we could move the attributes to the `exchange` tag?

We probably also need tests with multiple primary data and different bounds (also to test compatibility with the "preconditioner").

Reply:
> Could it also make sense to limit [secondary data](https://precice.org/configuration-acceleration.html#coupling-data-and-primary-data)? Then we could move the attributes to the exchange tag?

I put it in the acceleration scheme since the under-relaxation methods don't need this bounds to be given. So it's kind of specialized for QN-methods.

> We probably also need tests with multiple primary data and different bounds (also to test compatibility with the "preconditioner").

Yes, a more complex test makes sense. The current one is better for experiments, not to test all the functionalities ;)